packages/next/src/config/withFaust.ts

```diff
@@ -52,5 +52,9 @@ export function withFaust(
   nextConfig.redirects = () =>
     createRedirects(existingRedirects, previewDestination);
 
+  if(nextConfig.compress == null) {
```

Comment:
I'm not sure we would want to disable compression by default. It may lead to unintented consequences.

Atlas handles compression, but what about a platform that doesn't? How will that effect a user's build?

What is the downside of leaving `compress` in it's default state? Do the benefits outweight the possible unintended consequences?

Reply:
It doesn't affect the build. This is wire compression, so `Accept-Encoding` is ignored and assets aren't compressed on the fly. If means their server will perform better albeit at the expense of transfer size. Most (All?) CDNs that proxy data handle compression for you. So Vercel, Atlas, Netlify, etc are all going to be unaffected or benefit from this change. Self hosting could be worse technically if they don't use a CDN. But I think we should choose defaults that are generally beneficial. The logic allows folks to override if desired. Maybe we document this somewhere to make it more obvious.

Reply:
Leaving it as is just means servers potentially doing more work than needed if the edge/CDN could be handling compression.